sampler/src/AgateEnvironment/AgateEnvironment.js

```diff
@@ -129,22 +143,34 @@ const StorybookDecorator = (story, config) => {
 		}
 	};
 	const skinFromURL = getPropFromURL('skin');
-	const currentSkin = skinFromURL ? skinFromURL : 'gallium';
+	const currentSkin = skinFromURL ? skinFromURL : Config.defaultProps.skin;
 	const newSkin = (memory.skin !== currentSkin);
 	memory.skin = currentSkin;  // Remember the skin for the next time we load.
 	const accentFromURL = getPropFromURL('accent');
 	const highlightFromURL = getPropFromURL('highlight');
+	const allSkins = boolean('show all skins', Config);
+	const skinKnobs = {};
+	if (!allSkins) {
+		skinKnobs.skin = select('skin', skins, Config, currentSkin);
+	}
 
 	return (
 		<Agate
 			title={`${config.kind} ${config.story}`.trim()}
 			description={config.description}
-			skin={select('skin', skins, Config, currentSkin)}
-			skinVariants={boolean('Night Mode', Config, false) && 'night'}
+			{...skinKnobs}
+			skinVariants={boolean('night mode', Config) && 'night'}
 			accent={color('accent', (!newSkin && accentFromURL ? accentFromURL : defaultColors[currentSkin].accent), Config.groupId)}
 			highlight={color('highlight', (!newSkin && highlightFromURL ? highlightFromURL : defaultColors[currentSkin].highlight), Config.groupId)}
 		>
-			{sample}
+			<Scroller>
+				{allSkins ? Object.keys(skins).map(skin => (
+					<SkinFrame skin={skins[skin]} key={skin}>
```

Comment:
The current variant's background color doesn't appear to apply correctly.

Reply:
Are you sure? I was able to take a screenshot of it working and attach it to this PR.

Reply:
Do I need a special version of `agate`?  I'm just using `develop`.

Reply:
Gallium, day mode:
![image](https://user-images.githubusercontent.com/328450/72631461-42c37b00-3922-11ea-86e5-3fb078b3978a.png)

All skins, day mode:
![image](https://user-images.githubusercontent.com/328450/72631513-52db5a80-3922-11ea-84d1-f743f8ba97f4.png)

Reply:
All vs single:
<img width="460" alt="Day-mode-comparison" src="https://user-images.githubusercontent.com/386529/72631792-befd9480-3909-11ea-9959-7b7b84e14051.png">

Reply:
The real question is, why doesn't it work on your machine? Followed by, does it _only_ work on mine? We need a third perspective.

Reply:
On `develop`, I'm seeing what Roy is seeing; but it's fixed in `feature/gallium-sliderbutton`, as seen by Blake.

Reply:
Ah you're right! The `SliderButton` fix has corrections to the colors! Pointed out on that PR: https://github.com/enactjs/agate/pull/191#discussion_r367184783
Thanks @teckliew, I forgot all about that.